src/mumble/CoreAudio.cpp

```diff
@@ -491,8 +524,8 @@ CoreAudioOutput::CoreAudioOutput() {
 	desc.componentFlags = 0;
 	desc.componentFlagsMask = 0;
 
-	comp = FindNextComponent(NULL, &desc);
-	if (comp == NULL) {
+	comp = FindNextComponent(nullptr, &desc);
+	if (comp == nullptr) {
```

Comment:
I guess the "idiomatic" way of doing this check would be
```suggestion
	if (!comp) {
```

Ref.: https://stackoverflow.com/questions/3825668/checking-for-null-pointer-in-c-c